### ISSUE TITLE
RDKit Cartridge: qmol GiST support

### ISSUE THREAD

**Issue by mwojcikowski** [rdkit/rdkit#4463]
Somewhat related to #4442 - qmol columns cannot have GiST index added on them. As @greglandrum is making it easier to work with qmols in general, it would be great if we could accelerate them with proper indexing.

When I try to add an index I get the following error with RDKit 2021.03
```
ERROR: data type qmol has no default operator class for access method "gist"

**Comment by greglandrum** [comment]
We can certainly add this, but the fingerprint screening isn't often not super effective when working with qmols, so the performance gain won't be as dramatic as with normal mols.

**Comment by mwojcikowski** [comment]
I dug around yesterday and I think I found a way of adding it via rdkit.sql.in - will prepare a PR.

**Comment by greglandrum** [comment]
Fixed by #4470
